src/Http/Controllers/MeetupController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Controllers;
+
+use Blumilk\Meetup\Core\Http\Requests\StoreMeetupRequest;
+use Blumilk\Meetup\Core\Http\Requests\UpdateMeetupRequest;
+use Blumilk\Meetup\Core\Models\Meetup;
+
+class MeetupController extends Controller
+{
+    public function index()
+    {
+        $meetups = Meetup::latest()->with(["user"])->paginate(20);
+
+        return view("meetups.index")
+            ->with("meetups", $meetups);
+    }
+
+    public function create()
```

Comment:
Please add return types in all methods. Here it'll be probably `Illuminate\Contracts\View\View`.